cmd/immuclient/command/currentstatus.go

```diff
@@ -34,7 +34,7 @@ func (cl *commandline) currentState(cmd *cobra.Command) {
 			if err != nil {
 				cl.quit(err)
 			}
-			fmt.Fprintf(cmd.OutOrStdout(), resp+"\n")
```

Comment:
I think your initial approach of having a print function which adds a line break if not already included is good. But I'd do it as simple as possible i.e. adding if the last char is not a line break.
Note: with latest changes a unit testing is failing, probably because the error message differs from what's expected.

Reply:
Fixed the unit test. I was able to remove the `\n\n` branch but in the process I realized that if we have already done the work of putting right newline characters in place, we don't need to go through another step to check and append a newline character. I don't think it is a big deal in any case, though, I think we don't need to do it anymore. Let me know what you think.